scout/server/blueprints/alignviewers/controllers.py

```diff
@@ -0,0 +1,140 @@
+HG19REF_URL = (
```

Comment:
Moved a lot of track-related content to the controllers file, so the igv() method in the views is shorter and easier to read